composer-lock-diff

```diff
@@ -80,29 +81,44 @@ function version($pkg)
     return $version;
 }
 
+function sourceUrl($pkg) {
+    if (empty($pkg->source->url)) {
+        return '';
+    }
+
+    return str_replace('.git', '', $pkg->source->url);
+}
+
 function tableize($header, $data, $opts = array()) {
     if (empty($data)) return '';
 
     $opts = array_merge(array('capped' => true, 'joint' => '+'), $opts);
 
     if ($opts['no-links']) {
         array_walk($data, function(&$values) {
-            unset($values[2]);
+            unset($values[3]);
+            unset($values[4]);
         });
     } else {
         if (array_key_exists('url_formatter', $opts)) {
             $formatter = $opts['url_formatter'];
             array_walk($data, function(&$values) use ($formatter) {
-                $values[2] = call_user_func($formatter, $values[2], '...');
+                $values[0] = call_user_func($formatter, $values[4], $values[0]);
+                $values[3] = call_user_func($formatter, $values[2], '...');
+                unset($values[4]);
             });
         }
     }
 
     $titles = array($header, 'From', 'To');
-    if (! $opts['no-links']) array_push($titles, 'Compare');
-
-    $widths = array(maxLength(array_merge(array($header), array_keys($data))));
```

Comment:
Dropping this line is going to be trouble. If the package names are shorter than the title ("Production Changes" or "Dev Changes") then the widths will be off.

Reply:
```
- $widths[] = max(strlen($titles[$i + 1]), maxLength(array_map(function($k) use ($data, $i) { return $data[$k][$i]; }, array_keys($data))));
+ $widths[] = max(strlen($titles[$i]), maxLength(array_map(function($k) use ($data, $i) { return $data[$k][$i]; }, array_keys($data))));
```

It seems like this is no problem because I deleted `+ 1`